ja/mail/messenger/messenger.ftl

```diff
@@ -379,3 +385,14 @@ quick-filter-bar-show =
     .key = k
 
 ## OpenPGP
+
+openpgp-forget = OpenPGP のパスフレーズを忘れた
```

Comment:
`openpgp-forget = Forget OpenPGP passphrases`

Reply:
パスフレーズのタイムアウト機能を実装して、その有効時間 (2分) 以内にある場合に、強制タイムアウトを起こすボタンらしい。いわゆる「パニック」ボタンを想定していて、これを押すとパスフレーズの再入力が求められる、ようにしたいらしい。

> This string does NOT have UI yet.
> 
> It's a "reserve" string, for an additional functionality that I'd like to implement after the string freeze.
> 
> With the current implementation, the user has to enter the passphrase every single time when an operation with a secret key is performed (only for using secret keys where the user has opted in to use a used-defined passphrase).
> 
> I want to add a timeout mechanism in the near future, which will allow using those keys for a couple of minutes (configurable with hidden pref) without having to enter the passphrase again during this period of time.
> 
> In addition, I consider to also add an optional button on screen, which would be shown ONLY if there is currently at least one passphrase remembered by Thunderbird (after the user has unlocked using the passphrase), and the timeout period isn't over yet.
> 
> This would be a "panic" button, which the user could press if someone approaches the computer, or, if the user is simply done with using their OpenPGP secrets for now, and would like to lock Thunderbird again.

Reply:
保存されたパスフレーズを削除するものではない、と。

> 
> I wonder, pressing that button wouldn't REMOVE those passphrases from the keys that had a user-defined passphrase that. Maybe "Forget" is misleading?
> 
> Maybe it would be better to rather say "Lock".

Reply:
仮訳。

`openpgp-forget = 入力した OpenPGP のパスフレーズを時間切れにする`